### PR TITLE
Use Redux in React components.

### PR DESCRIPTION
**Pull request by lorenzoworx**
## Implemented Changes
In this pull request, I have implemented the following changes:
- Update the `initialState` variable in `booksSlice`.
- Display `books`, received from the slice, in a reusable component
- Dispatch actions using `useDispatch`
- Add a `<Button>` component responsible for adding a boo to the `books array`
- Add a `<Button>` component responsible for removing a book from the `books array`

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+      setTitle('');
+      setAuthor('');
+    }
+  };
+  return (
+    <section>
+      <h2>Add a new book</h2>
+      <form>
+        <input type="text" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Title" />
+        <input type="text" value={author} onChange={(e) => setTitle(e.target.value)} placeholder="Author" />
```
Comment (Shisui6) [inline src/components/AddBook.js]: There is a bug preventing books from being properly added to your application. Please double-check the methods you're passing to each input element when the value is changed🪲

File: src/components/Book.js [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
-        <button type="button">like</button>
-        <button type="button">Comment</button>
-        <button type="button">delete</button>
+const Book = ({ book }) => {
+  const dispatch = useDispatch();
+  return (
+    <li>
+      <div className="bookContainer">
+        <h2>{book.Title}</h2>
+        <p>{book.Author}</p>
```
Comment (Shisui6) [inline src/components/Book.js]: When you save a book, you save it with the title and author properties in lowercase. However, here, you are trying to access them using uppercase letters. This is causing your component not to display any data. Please update this component so the book data is displayed👍

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @lorenzoworx, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Good job connecting react and redux ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

- For future commits, Please remember to only capitalize the first letter of the commit message to adhere to best practices

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @lorenzoworx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Functioned Add & Remove ✔️
- Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**